charts/4allportal/values.yaml

```diff
@@ -281,6 +281,18 @@ webdav:
 
   podDisruptionBudget: true
 
+webdavEvent:
+  enabled: false
+  image:
+    registry: quay.io
+    repository: curl/curl
+    tag: latest
+
+  resources:
+    limits:
+      cpu: 2
+      memory: 256Mi
```

Comment:
I think two cores and 256Mi memory are way overkill :D. Maybe just one core and 32Mi memory, but shouldn't it be fine to just hardcode the limits? This container just needs to run tail, curl, and ash.

Reply:
Updated with following config:

```
          resources:
            requests:
              cpu: "100m"
              memory: 24Mi
            limits:
              cpu: "500m"
              memory: 32Mi
```